web-ng/src/app/components/form-field-editor/form-field-editor.component.ts

```diff
@@ -71,11 +72,27 @@ export class FormFieldEditorComponent implements OnInit, OnChanges, OnDestroy {
   @Output() delete = new EventEmitter();
   formOptions: MultipleChoice | undefined;
   selectFieldOptions: FieldTypeSelectOption[];
+  showOptionsAndActions = false;
+  private clickWasInside = false;
```

Comment:
Is this basically reimplementing focus/blur? Is there a reason not to rely on that instead?  If so, please leave a clarifying comment and perhaps rename `clickWasInside` to a name that describes the state of the component rather than how it go into that state (e.g., `selected`).